.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 lib
 dist
+package-lock.json
```

Comment:
This is only necessary to be backwards compatible with older versions of npm. If upgrading, the project should check this lockfile in.